react-app/src/library/types/consumed-headset-events.ts

```diff
@@ -23,9 +23,11 @@ type Events = {
     [HeadsetEvents.deviceRejectedCall]: RejectCallEventInfo;
     [HeadsetEvents.loggableEvent]: EventInfo;
     [HeadsetEvents.webHidPermissionRequested]: { callback: any };
-    [HeadsetEvents.deviceConnectionStatusChanged]: { currentVendor: VendorImplementation, isConnected: boolean, isConnecting: boolean};
+    [HeadsetEvents.deviceConnectionStatusChanged]: DeviceConnectionStatus;
```

Comment:
Just a note to remember to update the SDK docs when you pull this new version in.